script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -712,29 +768,34 @@ Anyone should be able to understand just how miserable of a conclusion that was.
 [sha:11e9e0e611329bdc4b6b97c075d2a33a4283b862]{
 -- Page 692, Offset 16528.
 -- 彼はこの時、初めて死を恐れた。
-It was at this time that he first began fearing death.
+It was at this time that he first began to fear death.
 }
 [sha:b0caaf2668ac0a6fb2073df1b7538e3cb9238ebc]{
 -- Page 692, Offset 16529.
 -- 死が怖い。無為が怖い。果てのない世界が怖い。これほど知識を得たというのに、私はまだ何一つとして知っていない。
-// trying to keep it third-person past tense here so ignoring watashi
-Death, futility, the infinite expanse of the world――they were frightening.
- He had gained so much knowledge, and yet, did not know a single thing.
+// P: Trying to keep it third-person past tense here so ignoring watashi
+// R: 無為 feels like a tricky one. Dictionary form idleness/inactivity doesn't really make sense to me here.
+// R: I've seen 無為に終わった translated as 'died without leaving a mark', so I guess P's 'futility' works well?
+// R: Unless we want to lean more into the oblivion, the insignificance compared to the 果てのない世界
+Death was frightening.
+ The futility of his actions was frightening.
+ The infinite expanse of the world was frightening.
+ He had gained so much knowledge, and yet, he did not know a single thing.
 }
 [sha:a9a8d92d3d681e1c4c6ead2fc849ef5bd9dd1c35]{
 -- Page 692, Offset 16530.
 -- 恐ろしいのは自らの消滅ではない。
-But what he feared was not his own demise,
+But what terrified him was not his own demise―――
 }
 [sha:a94b19b2a5a6589cbfbea9d245f33a7c92b6b9c7]{
 -- Page 692, Offset 16531.
 -- 自らの<目|い><的|み>が砕け散るのが、こんなにも恐ろしいのだ。
-it was seeing his own purpose fall apart.
+It was seeing his purpose fall apart.
 }
 [sha:191972a4b6c821b2d0b7f380c7e71bbfff8d50a1]{
 -- Page 693, Offset 16532.
 -- 思考を切り替える必要があった。
-He needed to alter his thinking.
+He needed to frame things in a new light.
```

Comment:
I don't think framing things in a new light is quite appropriate for what Roa does here.

Reply:
revert